sequencer/src/bin/deploy.rs

```diff
@@ -209,16 +212,16 @@ struct Options {
     #[clap(long, env = "ESPRESSO_SEQUENCER_EPOCH_START_BLOCK")]
     epoch_start_block: Option<u64>,
     /// The initial supply of the tokens.
-    #[clap(long, env = "ESP_TOKEN_INITIAL_SUPPLY", default_value_t = U256::from(3590000000u64))]
-    initial_token_supply: U256,
+    #[clap(long, env = "ESP_TOKEN_INITIAL_SUPPLY")]
+    initial_token_supply: Option<U256>,
```

Comment:
I think I'm a bit lost as to why we're making this change to remove the defaults.

Reply:
The goal behind removing the default value is to force explicit configuration, either via CLI or environment variable. The idea is that the config should always be correct and intentional, and we shouldn’t fall back to a hardcoded default that might silently cause incorrect behavior. Can revert if you don't agree

Reply:
I think that's fine. In general I have a bit of a hard time imagining needing configurability for the token name and symbol. If it's not needed it's usually better to not have it.

I think I might be more lost with the purpose of the current V2 contract, since it only overrides the token name with a hardcoded value in the contract. W haven't deployed V2 on decaf and will deploy V1 on mainnet

So I'm wondering if we should make the V2 a placeholder contract for now and develop the rewards withdrawal into it that contract later,

The PR can just mostly stay as it is, it's nice that we have the infra to deploy the new contract already.